### PR TITLE
Update integrate-bazaarvoice.md

### PR DESCRIPTION
**Pull request by jeremyfourna**
## PR Description

Add the trusted hosts' changes that ACP customers will need to do.

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by k4emic — DISMISSED** [review]

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ The BazaarVoice app requires the following Spryker modules:
 
 ## Integration
 
+## 1. Connect Bazaarvoice
```
Comment (VadymSachenko) [inline docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:23]: Are we integrating or connecting it?

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md [hunk 2]
```diff
@@ -40,5 +42,29 @@ You can do the administration work on the Bazaarvoice reviews from the [Bazaarvo
 
 {% endinfo_block %}
 
+## 2. Whitelist Bazaarvoice Domain
```
Comment (VadymSachenko) [inline docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:45]:
```suggestion
## 2. Add Bazaarvoice domain to your allowlist
```

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md [hunk 3]
```diff
@@ -40,5 +42,29 @@ You can do the administration work on the Bazaarvoice reviews from the [Bazaarvo
...
+
+To enable your customers to leave reviews on your products, you must add Bazaarvoice domain inside your **Content Security Policy** whitelist. To do that please change your `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.
```
Comment (VadymSachenko) [inline docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:47]:
```suggestion
To enable your customers to leave reviews on your products, you must add the Bazaarvoice domain inside your **Content Security Policy** allowlist. To do that, change your `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.
```

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md [hunk 4]
```diff
@@ -40,5 +42,29 @@ You can do the administration work on the Bazaarvoice reviews from the [Bazaarvo
...
+
+Changes required in the `deploy.yml` file.
```
Comment (VadymSachenko) [inline docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:49]:
```suggestion
In the `deploy.yml` file, introduce the required changes: 
```

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md [hunk 5]
```diff
@@ -40,5 +42,29 @@ You can do the administration work on the Bazaarvoice reviews from the [Bazaarvo
...
+    SPRYKER_AOP_APPLICATION: '{
+      "APP_DOMAINS": [
+        "*.bazaarvoice.com",
+        ...
+      ],
+      ...
+    }'
+```
+
+Alternatively, you may whitelist the domain from the `config/Shared/config_default.php` file. If you updated the `deploy.yml` file, then this step can be ignored.
```
Comment (VadymSachenko) [inline docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:63]:
```suggestion
Alternatively, you may add the domain to the allowlist from the `config/Shared/config_default.php` file. If you updated the `deploy.yml` file, then this step can be ignored.
```

**Review by VadymSachenko — CHANGES_REQUESTED** [review]

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md (hunk 1 above)
Comment (jeremyfourna) [reply docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md]: In ACP we are "Connecting" apps lol

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**